package.json

```diff
@@ -74,6 +75,8 @@
     "typescript": "^5.1.6"
   },
   "dependencies": {
+    "@tanstack/react-router": "^1.45.4",
+    "@tanstack/start": "^1.45.4",
```

Comment:
This won't end up here. It would likely end up in a `tanstack-auth` package along with the `TanStackAuthenticator` class.